modules/git-flow-hooks-config.sh.dist

```diff
@@ -3,3 +3,5 @@ VERSION_BUMPLEVEL_HOTFIX="PATCH"
 VERSION_BUMPLEVEL_RELEASE="MINOR"
 VERSION_SORT="/usr/bin/sort -V"
 VERSION_TAG_PLACEHOLDER="%tag%"
+USE_CHANGELOG=false
+CHANGELOG_FILE_NAME="CHANGELOG"
```

Comment:
I was thinking you'd take over the feature if you wanted to include it.
Writing shell scripts isn't what I do, I just wanted this feature for my projects.

But if you want me to refine it first, I can do that.